apps/web/src/components/Sidebar/DocumentTree/index.tsx

```diff
@@ -0,0 +1,80 @@
+'use client'
```

Comment:
Don't put components outside where they're used. This sidebar is only going to be used in a `project/prompts` route. So make this

```javascript
apps/
  (private)
    projects/
      [:id]/
        page.tsx
        _components/
          Sidebar/index.tsx
```

Let's collocate as much as possible. Folders prefixed by `_` are ignored by nextjs router